src/lib/es2021.intl.d.ts

```diff
@@ -16,9 +21,4 @@ declare namespace Intl {
         dayPeriod?: "narrow" | "short" | "long";
         fractionalSecondDigits?: 0 | 1 | 2 | 3;
     }
-
-    interface NumberFormat {

```

Comment:
Could you perhaps move this to esnext.intl.d.ts?

Reply:
Thanks for the review!
I've fixed it.